forge/config/index.js

```diff
@@ -48,13 +48,18 @@ module.exports = fp(async function (app, opts, next) {
     if (process.env.npm_package_version) {
         ffVersion = process.env.npm_package_version
         // npm start
-        app.log.info(`FlowForge v${process.env.npm_package_version}`)
     } else {
         // everything else
         const { version } = require(path.join(module.parent.path, '..', 'package.json'))
         ffVersion = version
-        app.log.info(`FlowForge v${version}`)
     }
+    try {
+        fs.statSync(path.join(__dirname, '..', '..', '.git'))
```

Comment:
`git describe` is what you're looking for @sammachin

Reply:
this code was lifted out of the Node-RED core to keep the design pattern consistent